tailwind.config.ts

```diff
@@ -10,6 +10,9 @@ export default {
         warm: '#E11D48',
         grayBackground: 'rgba(217, 217, 217, 0.25);',
       },
+      gridTemplateColumns: {
+        'auto-fill-events': 'repeat(auto-fill, minmax(22rem, 1fr))',
```

Comment:
Haven't seen this before. Seems a bit odd that we define a custom property in the config file for a one-time usage? Could we not write this inline in the component? Maybe `grid-cols-[repeat(auto-fill, minmax(22rem, 1fr))]`?

Reply:
-> https://tailwindcss.com/docs/adding-custom-styles#using-arbitrary-values

Reply:
ah, good point, originally was thought all grids that uses event card could also use this property, but now I think about it, this property does have a pretty limited use case;

will change it